charts/kyverno/templates/deployment.yaml

```diff
@@ -104,6 +104,10 @@ spec:
                 fieldPath: metadata.namespace
           - name: KYVERNO_SVC
             value: {{ template "kyverno.serviceName" . }}
+          {{- with .Values.envVars }}
+          env:
```

Comment:
env key for this container is already declared on line 98.

Reply:
i'll clean these up with the new split in envVars